runtime/README.md

```diff
@@ -1030,29 +1443,201 @@ The DPE `GET_CERTIFICATE_CHAIN` command shall return the following certificates:
+| Validity                       | notBefore    | notBefore from firmware manifest                                    |
+|                                | notAfter     | notAfter from firmware manifest                                     |
+| Subject Name                   | CN           | Caliptra DPE Leaf                                                   |
+|                                | serialNumber | SHA384 hash of Subject public key                                   |
+| Subject Public Key Info        | Algorithm    | ecdsa-with-SHA384                                                   |
+|                                | Parameters   | Named Curve = prime384v1                                            |
+|                                | Public Key   | DPE Alias Public Key value                                          |
+| Signature Algorithm Identifier | Algorithm    | ecdsa-with-SHA384                                                   |
+|                                | Parameters   | Named Curve = prime384v1                                            |
+| Signature Value                |              | Digital signature for the certificate                               |
+| KeyUsage                       | keyCertSign  | 1                                                                   |
+| Basic Constraints              | CA           | False                                                               |
+| Policy OIDs                    |              | id-tcg-kp-attestLoc                                                 |
+| tcg-dice-MultiTcbInfo\*        | FWIDs        | [0] "Journey" TCI Value                                             |
+|                                |              | [1] "Current" TCI Value. Latest `INPUT_DATA` made by DeriveContext. |
+|                                | Type         | 4-byte TYPE field of TCI node                                       |
+|                                | VendorInfo   | Locality of the caller (analog for PAUSER)                          |
 
 \*MultiTcbInfo contains one TcbInfo for each TCI Node in the path from the
 current TCI Node to the root. Max of 32.
+
+
+## Manifest-Based Image Authorization (1.2)
+
+Caliptra's goal is to enable integrators and vendors to meet standard security requirements for creating cryptographic identity and securely reporting measurements through DICE and DPE Certificate chains and Caliptra-owned private-public key pairs. In addition, Caliptra 1.0 provides an `ECDSA384_SIGNATURE_VERIFY` command to enable an SoC RoT to verify its own FW signatures so that it can develop an SoC secure boot using Caliptra cryptography. Caliptra 1.1 expanded the verify command to a PQC-safe `LMS_SIGNATURE_VERIFY` command. In each of these cases, it is left up to the vendor to ensure that they build a secure environment for introducing and verifying FW integrity and authenticity and then executing mutable FW.
```

Comment:
nit: integrator and vendor generally refer to the same entity in Caliptra documentation.